addons/binding/pom.xml

```diff
@@ -120,12 +120,13 @@
     <module>org.openhab.binding.tplinksmarthome.test</module>
     <module>org.openhab.binding.urtsi</module>
     <module>org.openhab.binding.velbus</module>
+    <module>org.openhab.binding.valloxmv</module>
```

Comment:
Please put it on place up.

Reply:
To have the binding being picked up by the distro, you furthermore need to add it to the feature.xml, again at the alphabetically correct position. If you have a dependency on some transport bundle (e.g. upnp, mdns or serial), make sure to add a line for this dependency as well (see the other bindings as an example).

Reply:
Changed